medicinal_plants/lib/src/screens/detail-screen.dart

```diff
@@ -0,0 +1,173 @@
+// ignore_for_file: avoid_unnecessary_containers
+
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:medicinal_plants/src/constants/colors.dart';
+
+class DetailScreen extends StatelessWidget {
+  const DetailScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    double width = MediaQuery.of(context).size.width;
+    double height = MediaQuery.of(context).size.height;
+
+    return Scaffold(
+      body: Stack(
```

Comment:
use appBar from scaffold,

as for the image, i think it would return the same result without using the Stack, just a column will do